### PR TITLE
Correct height for media gallery images

### PR DESCRIPTION
**Pull request by Nazar65**
### Description (*)

Remove image height usage from Paragraph as this is not necessary.

### Fixed Issues (if relevant)

N/A

### Manual testing scenarios (*)
1. Verify Media gallery images displayed correctly in all sizes.
2. Define any custom sizes in di.xml Magento/Cms/etc/di.xml
```
    <type name="Magento\Cms\Model\Wysiwyg\Images\Storage">
        <arguments>
            <argument name="resizeParameters" xsi:type="array">
                <item name="height" xsi:type="number">75</item>
                <item name="width" xsi:type="number">100</item>
            </argument>
   </type>
```
Verify All images displayed correctly

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Cms/view/adminhtml/templates/browser/content/files.phtml [hunk 1]
```diff
@@ -7,25 +7,29 @@
...
+?>
     <div
         data-row="file"
         class="filecnt"
         id="<?= $block->escapeHtmlAttr($block->getFileId($file)) ?>"
         data-size="<?= $block->escapeHtmlAttr($file->getSize()) ?>"
         data-mime-type="<?= $block->escapeHtmlAttr($file->getMimeType()) ?>"
     >
-        <p class="nm" style="height:<?= $block->escapeHtmlAttr($_height) ?>px;">
+        <p class="nm" style="min-height: 72px;">
```
Comment (sivaschenko) [inline app/code/Magento/Cms/view/adminhtml/templates/browser/content/files.phtml]: Can you please avoid the inline css

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @Nazar65 , thanks for the pull request! Please see my comment

File: app/code/Magento/Cms/view/adminhtml/templates/browser/content/files.phtml (hunk 1 above)
Comment (Nazar65) [reply app/code/Magento/Cms/view/adminhtml/templates/browser/content/files.phtml]: ok :+1:

Comment (krzksz) [reply app/code/Magento/Cms/view/adminhtml/templates/browser/content/files.phtml]: We can also use `height` attribute if there is no other option.

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-7501](https://jira.corp.magento.com/browse/ENGCOM-7501) has been created to process this Pull Request
:eight_spoked_asterisk: @sivaschenko, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by sivaschenko** [comment]
The automated test coverage is not applicable as this is a style change

**Comment by engcom-Bravo** [comment]
:heavy_check_mark: QA Passed

**Comment by engcom-Echo** [comment]
Failed functional tests not related to the changes in this PR

**Pull request merged**